release/test/regenerate_releases_json_test.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+source ../_liferay_common.sh
+source ../_releases_json.sh
+
+function merge_json_snipets_test {
+    local earliest_url
+    local latest_url
+
+   	if [ "${1}" == "dxp" ]
+    then
+        earliest_url="$(jq -r '.[0].url' < "$(find ./20*dxp*.json | head -n 1)")"
+        latest_url="$(jq -r '.[0].url' < "$(find ./20*dxp*.json | sort -r | head -n 1)")"
+    else
+        earliest_url="$(jq -r '.[0].url' < "$(find ./20*ga*.json | head -n 1)")"
+        latest_url="$(jq -r '.[0].url' < "$(find ./20*ga*.json | sort -r | head -n 1)")"
```

Comment:
@lucasmiranda0, let's remove this part, for now, since this code isn't being used in the test scope